tailwind.config.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  content: ['./src/**/*.{html,js}'],
+  theme: {
+    extend: {},
+  },
+  plugins: [],
+  corePlugins: {
+    preflight: false,
```

Comment:
This is totally fine to include for now but we may have issues with Tailwind without preflight. I seem to recall that some of the Tailwind style expect the resets that come with preflight but if that's not the case then this can certainly stay long-term